### PR TITLE
add new achievemens - new branch for achievements 2024

### PR DESCRIPTION
**Pull request by PiechZ**
Please review my new achievements.

### CODE REVIEW

File: sql_prep/models/achievements/agg__napul_cesty.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
+WITH ib_base AS (
+    SELECT * FROM {{ ref('base__debater_ib') }}
+),
+
+aggregate_ib AS (
+    SELECT
+        clovek_id,
+        ANY_VALUE(school_year) as school_year,
```
Comment (shippy) [inline sql_prep/models/achievements/agg__napul_cesty.sql:8]: Why the ANY_VALUE? Wouldn't it make sense to skip it altogether if you don't want to aggregate by it?

**Review by shippy — CHANGES_REQUESTED** [review]
Question about the use of ANY_VALUE (I think that could kill some achievements at the end of achievements__final given the school year limitation - if anything, MAX might make more sense) + I assume I should make the images?

**Review by PiechZ — COMMENTED** [review]
I fixed these two achievements according to your advice.

**Comment by shippy** [comment]
Fun fact: Windows and Unix are different in that one is case-sensitive (Unix) and the other is not (Windows). So I'm guessing the capital-letters name doesn't make a difference on Windows in terms of `ref`, but does on Unix/MacOS.

**Pull request merged**